cmd/kubeadm/app/cmd/phases/bootstraptoken.go

```diff
@@ -107,14 +264,95 @@ func NewSubCmdNodeBootstrapTokenPostCSRs(kubeConfigFile *string) *cobra.Command
 func NewSubCmdNodeBootstrapTokenAutoApprove(kubeConfigFile *string) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "allow-auto-approve",
-		Short: "Configure RBAC rules to allow the csrapprover controller automatically approve CSRs from a node bootstrap token.",
+		Short: "Configures RBAC rules to allow the csrapprover controller automatically approve CSRs from a node bootstrap token",
+		Long:  nodeAutoApproveLongDesc,
 		Run: func(cmd *cobra.Command, args []string) {
 			client, err := kubeconfigutil.ClientSetFromFile(*kubeConfigFile)
 			kubeadmutil.CheckErr(err)
 
+			// Create RBAC rules that makes the bootstrap tokens able to get their CSRs approved automatically
 			err = node.AutoApproveNodeBootstrapTokens(client)
 			kubeadmutil.CheckErr(err)
+
+			// Create/update RBAC rules that makes the nodes to rotate certificates and get their CSRs approved automatically
+			err = node.AutoApproveNodeCertificateRotation(client)
+			kubeadmutil.CheckErr(err)
 		},
 	}
 	return cmd
 }
+
+func addBootstrapTokenFlags(flagSet *pflag.FlagSet, cfg *kubeadmapiext.MasterConfiguration, cfgPath, description *string, usages, extraGroups *[]string, skipTokenPrint *bool) {
```

Comment:
from here and below, the code seems to be unnecessarily duplicated between `kubeadm phase bootstrap-token` and  `kubeadm token create` I think the code here should match each other exactly and be re-used.